timelock-server/src/integTest/resources/lock/one-default-one-higher-explicit-timeout.yml

```diff
@@ -0,0 +1,23 @@
+clients:
+  - test
+  - test2
+
+cluster:
+  localServer: localhost:8421
+  servers:
+    - localhost:8421
+
+algorithm:
+  type: paxos
+
+timeLimiter:
+  enableTimeLimiting: true
+  blockingTimeoutErrorMargin: 0.03
+
+server:
+  applicationConnectors:
+    - type: http
+      port: 8080
+    - type: https
+      port: 808
+      idleTimeout: 90 seconds
```

Comment:
Yes... I wouldn't mind bumping it in our internal recommended configurations. (The normal defaults are a Dropwizard/Jetty thing)